examples/versioning/versioned_function.py

```diff
@@ -64,12 +64,12 @@ def abs_diff(a: pd.Series, b: pd.Series) -> pd.Series:
 # Next, we implement the new version of the `abs_diff` and mark it as version 1.1.
 #
 
-from indsl.resample import reindex  # noqa
+from indsl.resample import reindex_v1  # noqa
 
 
 @versioning.register(version="1.1")  # type: ignore
 def abs_diff(a: pd.Series, b: pd.Series) -> pd.Series:
-    a, b = reindex(a, b)
+    a, b = reindex_v1.reindex(a, b)
```

Comment:
```suggestion
    a, b = reindex(a, b)
```
I believe the newest reindex version should be used here.